packages/react/src/components/IconButton/index.tsx

```diff
@@ -23,18 +24,25 @@ const IconButton = forwardRef<HTMLButtonElement, IconButtonProps>(
       label,
       tooltipPlacement = 'auto',
       className,
+      variant = 'secondary',
       ...other
     }: IconButtonProps,
-    ref
+    ref: any
```

Comment:
❗ add a prop ref type here

Reply:
Sorry, not sure why that was still there. It should be an inferred type.